packages/app/src/components/Root/LogoFull.tsx

```diff
@@ -1,30 +1,36 @@
 import React from 'react';
-import { makeStyles } from '@material-ui/core';
+// import { makeStyles } from '@material-ui/core';
 
-const useStyles = makeStyles({
-  svg: {
-    width: 'auto',
-    height: 30,
-  },
-  path: {
-    fill: '#7df3e1',
-  },
-});
+import MyCustomLogoFull from './logo/Tux.svg.png'
 const LogoFull = () => {
-  const classes = useStyles();
```

Comment:
When pushing code to main, make sure to delete all the commented code to keep the code clean.